content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>I think this should read: "<strong>Identifying</strong> regions of interest (...) can be useful"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Good catch, fixed it! Thanks